src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -36,27 +36,34 @@
 
 import wx
 
-from odemis import model, dataio
+from odemis import dataio, model
 from odemis.acq import acqmng, stream
-from odemis.acq.stream import FluoStream, StaticStream, BrightfieldStream
+from odemis.acq.feature import acquire_at_features, add_feature_info_filename
+from odemis.acq.stream import BrightfieldStream, FluoStream, StaticStream
 from odemis.gui import conf
 from odemis.gui.cont.acquisition._constants import VAS_NO_ACQUISITION_EFFECT
-from odemis.gui.cont.acquisition.overview_stream_acq import OverviewStreamAcquiController
+from odemis.gui.cont.acquisition.overview_stream_acq import (
+    OverviewStreamAcquiController,
+)
 from odemis.gui.util import call_in_wx_main, wxlimit_invocation
-from odemis.gui.util.widgets import ProgressiveFutureConnector, VigilantAttributeConnector
+from odemis.gui.util.widgets import (
+    ProgressiveFutureConnector,
+    VigilantAttributeConnector,
+)
 from odemis.gui.win.acquisition import ShowAcquisitionFileDialog
 from odemis.util import units
 from odemis.util.comp import generate_zlevels
-from odemis.util.dataio import splitext
-from odemis.util.filename import guess_pattern, create_filename, update_counter
-
+from odemis.util.filename import create_filename, guess_pattern, update_counter
 
 # constants for the acquisition future state of the cryo-secom
 ST_FINISHED = "FINISHED"
 ST_FAILED = "FAILED"
 ST_CANCELED = "CANCELED"
 
 
+# tmp flag for odemis advanced mode
```

Comment:
What exactly is advanced mode and what extra functionality do you get when it is activated?

Reply:
its a temporary flag for a licenced version